### PR TITLE
This pull request is to add my first capstone project 

### PR DESCRIPTION
**Pull request by DiegoMero**
This is my first capstone project, I used HTML, CSS, and javascript to do it, it is about a race in the Mario World Universe.

Here is the required video:
https://www.loom.com/share/1041c84108a9498786380a3a21081c18

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,29 @@
 # Super-Mario-Run
 This webpage is to represent a race in the Mario world.
```
Comment (tufoinnkuo10) [inline README.md]: - [ ] kindly add a good description of what this project is all about

File: README.md [hunk 2]
```diff
@@ -1,2 +1,29 @@
...
+- GitHub: [@githubhandle](https://github.com/githubhandle)
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+## 🤝 Contributing:
+ - Contributions, issues, and feature requests are welcome!
+ - Feel free to check the issues page.
+## Show your support:
+ - Give a ⭐️ if you like this project!
+## Acknowledgments:
+ Thanks to microverse ⭐️.
```
Comment (tufoinnkuo10) [inline README.md]: - [ ] the acknowledgments should give appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

File: README.md [hunk 3]
```diff
@@ -1,2 +1,29 @@
 # Super-Mario-Run
 This webpage is to represent a race in the Mario world. 
+## Built With:
+ - HTML
+ - CSS
+ - JavaScript
+## Prerequisites:
```
Comment (tufoinnkuo10) [inline README.md:9]: - [ ] kindly make use of github pages and ensure that the project is deployed and add the link in the readme file

File: about.html [hunk 4]
```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
```
Comment (tufoinnkuo10) [inline about.html:5]: - [ ] kindly make your mobile popup menu full height. also, the about link is highlighted without me touching it or without me being on the about page. it kind of makes it different from the others. kindly fix that.
Screenshot (178)_LI

File: index.html [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+            <div class="menu-icon-placeholder">
+                <div class="menu-button" id="popup-menu"></div>
+            </div>
+        </div>
+    </nav>
+    <main>
+        <section id="introduction">
+            <h3 class="mini-title">It's Mario Time!</h5>
+            <h1 class="title">SUPER MARIO RUN 2022 EDITION</h1>
+            <p class="event-description">An incredible race where the goal is to reach the finish line before everyone else and defeat the evil Bowser.</p>
```
Comment (tufoinnkuo10) [inline index.html:29]: -[ ] kindly use thesame font size here

File: index.html [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+                    <h3 class="activity-title">Boss</h3>
+                    <p class="activity-description">Are you ready for the final challenge? Fortune and glory, kid.</p>
+                </div>
+            </div>
+            <button class="join-button" type="button">Join Super Mario Run 2022</button>
+        </section>
+        <section class="characters" id="feature-characters">
+        </section>
+        <section id="partners">
+            <h2 class="partners-title">Partner</h2>
```
Comment (tufoinnkuo10) [inline index.html:68]: - [ ] you are missing the partner section in the mobile about page.
your version 👇🏾 
Screenshot (180)
Behance version 👇🏾 
Screenshot (181)_LI

File: index.html [hunk 7]
```diff
@@ -0,0 +1,85 @@
...
+            <div class="divisor"></div>
```
Comment (tufoinnkuo10) [inline index.html:69]: - [ ] please the about page footer should have a white background

File: index.html [hunk 8]
```diff
@@ -0,0 +1,85 @@
...
+    <title>Super Mario Run</title>
+    <link rel="stylesheet" href="styles.css">
+    <link href="https://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato&display=swap" rel="stylesheet">      
+</head>
+<body>
+    <nav>
+        <div class="navbar">
```
Comment (tufoinnkuo10) [inline index.html:16]: - [ ] please make the nav bar links of program, join, and sponsors(Featured Characters) to take you to their various page sections.

File: popupMenu.js [hunk 9]
```diff
@@ -0,0 +1,50 @@
+const hamburger = document.getElementById('popup-menu');
```
Comment (tufoinnkuo10) [inline popupMenu.js:1]: - [ ] according to the rules and design, your mobile hamburger should be on the left. kindly fix that, you can exchange it with your logo. 
Screenshot (182)_LI

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @MoRaad97 

Nice work so far, you are almost there. There are some issues that you still need to work on to complete the project.

## To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated
✅ responsive
✅design well followed
✅ mobile and desktop works well

## Required Changes ♻️♻️
Check the comments under the review
- great job with the good video explanation

tufoin

Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by rotimiazeez — APPROVED** [review]
# Hi @DiegoMero 👏🏽, 

### Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**